CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ the detailed section referring to by linking pull requests or issues.
 
 #### Minor
```

Comment:
![image](https://github.com/sovity/edc-ui/assets/30114507/ce663b38-cf36-4a36-8107-f18469bb485f)

without focusing, the placeholder does not show, maybe adding a description text could be nice.